toolkit/offchain/src/csl.rs

```diff
@@ -14,6 +14,36 @@ use ogmios_client::{
 use sidechain_domain::{AssetId, NetworkType, UtxoId};
 use std::collections::HashMap;
 
+#[derive(Debug, Clone, PartialEq)]
+pub struct HelperTransaction {
```

Comment:
This file has a convention of using extension traits, https://rust-lang.github.io/rfcs/0445-extension-trait-conventions.html please keep using them